releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,17 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.1.0-preview - Jun XX, 2023
+
+### New Features
+
+* Dynamic configuration support, support refreshing the data of ConfigMap or Secret by watching the sentinel key periodically.
+* HMAC authentication support.
+
+### Bug Fixes
+
+* Fixed an issue happens while multiple selectors are used.

```

Comment:
How does the issue appear to a user?

Reply:
The issue will occur to the user when all the following situations meet
1. multiple selectors are specified
2. Any of the selectors returns an empty set of key-value
3. The selector which returns empty set is not the last one in the selector list

It will cause the selectors after the selector which returns empty set won't work as expected. It will stop at the empty set selector